src/themes/teams/components/Radio/radioStyles.ts

```diff
@@ -0,0 +1,36 @@
+import { IComponentPartStylesInput, ICSSInJSStyle } from '../../../../../types/theme'
+
+const radioStyles: IComponentPartStylesInput = {
+  root: ({ props, variables }): ICSSInJSStyle => {
+    return {
+      display: 'inline-flex',
+      position: 'relative',
+      alignItems: 'center',
+      justifyContent: 'flex-end',
```

Comment:
could you, please, suggest where this is necessary?

Reply:
![image](https://user-images.githubusercontent.com/1826985/44348201-117b1d00-a49a-11e8-9800-06f01c9eb8ee.png)

Reply:
but the justifyContent will be removed from here in next update